recipes/pydusa/meta.yaml

```diff
@@ -3,8 +3,8 @@ package:
     version: 1.15
 
 source:
-    url: https://github.com/cryoem/pydusa/archive/master.tar.gz
-    fn: pydusa-master.tar.gz
+    url: https://github.com/cryoem/pydusa/archive/{{environ.get('GIT_PYDUSA_VERSION', 'v20170831')}}.tar.gz
```

Comment:
`GIT_PYDUSA_VERSION` needs to be set in `build_pydusa_numpy.sh` before `conda-build` is called and needs to be whitelisted in this file, `meta.yaml`.